src/oaklib/utilities/subsets/value_set_expander.py

```diff
@@ -183,6 +183,8 @@ def _generate_permissible_value(
         enum_definition: EnumDefinition = None,
     ) -> PermissibleValue:
         definition = oi.definition(curie)
+        # \n can break some downstream tooling like LinkML's gen-pydantic (v1.6.6)
```

Comment:
I think this should be fixed in the tooling, as newlines are valid (if unusual) in textual elements. But we can merge this for now